readme.md

```diff
@@ -76,109 +266,206 @@ specify the `--config` argument to Bazel with one of the following values:
+
+This example relays incoming corrections data to a receiver over a serial connection. The receiver should be configured
+to send NMEA `$GPGGA` position updates over the connection. They will then be forwarded to Polaris to associate the
+receiver with an appropriate corrections stream.
+
+To run the application, run the following command:
+```
+bazel run //examples:serial_port_example -- --polaris_api_key=<POLARIS_API_KEY> --device=/dev/ttyACM0
+```
 
-### NTRIP Proxy Example ###
+#### NTRIP Proxy Example ####
 
-The example binary `ntrip_example` proxies Polaris RTK corrections as a simple NTRIP caster for use with devices that take NTRIP corrections.
+Receive incoming Polaris corrections data and relay it as a simple NTRIP caster for use with devices that take NTRIP
+corrections. Receiver positions sent to the NTRIP server (via a NMEA `$GPGGA` message) will be forwarded to Polaris to
+associate the receiver with an appropriate corrections stream.
 
-Any receiver that uses a NTRIP connection can simply connect to the computer running this application on port 2101 using the following command:
+For example, to run an NTRIP proxy on TCP port 2101 (the standard NTRIP port), run the following command:
 ```
-bazel run -c opt examples/ntrip:ntrip_example -- --polaris_api_key=${MY_POLARIS_APP_KEY} 0.0.0.0 2101 examples/ntrip
+bazel run -c opt examples/ntrip:ntrip_example -- --polaris_api_key=<POLARIS_API_KEY> 0.0.0.0 2101 examples/ntrip
 ```
 
-Your receiver should be configured to send NMEA GGA messages at a regaulr cadence (usually 1hz) so it is properly receiving corrections for the right region. The default mount point for the examples is `/Polaris` and no username or password is required.
+Any GNSS receiver that supports an NTRIP connection can then connect to the computer running this application to receive
+corrections, connecting to the NTRIP endpoint `/Polaris`. The receiver should be configured to send NMEA `$GPGGA`
+messages at a regular cadence (typically 1hz).
+
+#### Septentrio Receiver Example ####
 
-The examples directory also contains a simple `ntrip-proxy.service` that can be used to turn the example into a standard Linux service.
+Similar to the [Generic Serial Receiver Example](#generic-serial-receiver-example), this application relays RTK
+corrections to a Septentrio GNSS receiver over a serial connection, and forward positions from the receiver to Polaris.
+Positions are translated from Septentrio Binary Format (SBF) as needed.
 
-### Septentrio Receiver Example ###
 
-The example binary `septentrio_example` streams RTK corrections to a Septentrio receiver over serial and reports back the receivers PVT message to the console. This has been tested on Septentrio's [AsteRx-m2](https://www.septentrio.com/en/products/gnss-receivers/rover-base-receivers/oem-receiver-boards/asterx-m2). The example can be easily modified for receivers that may receive corrections over other interfaces.
+This has been tested on Septentrio's
```

Comment:
We should update this to say Mosaic...the m2 is now EOL.

Reply:
actually lets open a ticket to do that later, the scope is a little larger given the configs and other parts.